internal_packages/composer-emoji/lib/emoji-composer-extension.jsx

```diff
@@ -205,17 +206,13 @@ class EmojiComposerExtension extends ComposerExtension {
       }
     }
     const emojiChar = emoji.get(emojiName);
-    if (process.platform === "darwin" && missingEmojiList.indexOf(emojiName) !== -1) {
-      const html = `<img
-                      class="missing-emoji ${emojiName}"
-                      src="images/composer-emoji/missing-emoji/${emojiName}.png"
-                      width="14"
-                      height="14"
-                      style="margin-top: -5px;">`;
-      editor.insertHTML(html, {selectInsertion: false});
-    } else {
-      editor.insertText(emojiChar);
-    }
+    const html = `<img
+                    class="emoji ${emojiName}"
```

Comment:
Oh, yeah, I already changed that! It should be `getImagePath`. Easier to determine the platform and send the correct file path from `EmojiStore`, I think, instead of doing it across three or four different files.